src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Profile.cshtml

```diff
@@ -0,0 +1,116 @@
+﻿@page "{UserName?}"
+@model APIViewWeb.Pages.Assemblies.ProfileModel
+@using APIViewWeb.Helpers
+@using APIViewWeb.Models
+@{
+    if(Model.User.GetGitHubLogin().Equals(Model.userProfile.UserName)) 
+    {
+        ViewData["Title"] = "Your profile";
+    }
+    else 
+    {
+        ViewData["Title"] = Model.userProfile.UserName;
+    }
+    TempData["UserPreference"] = PageModelHelpers.GetUserPreference(Model._preferenceCache, User.GetGitHubLogin()) ?? new UserPreferenceModel();
+
+    List<string> themes = new List<string>()
+    {
+        "light-theme",
+        "dark-theme",
+        "dark-solarized-theme"
+    };
+    string[] Languages = { "c", "c#", "c++", "go", "java", "javascript", "python", "protocol", "swift" };
+    var userPreference = (TempData["UserPreference"] as UserPreferenceModel) ?? new UserPreferenceModel();
+    var theme = userPreference.Theme ?? "light-theme";
+}
+@section Scripts
+{
+    <script type="text/javascript" src="@Url.Content("~/userProfile.js")" asp-append-version="true"></script>
+}
+<div class="container-fluid py-2">
+    @if(Model.userProfile == null) 
+    {
+        <h1> User profile does not exist. </h1>
+    }
+    else if (Model.userProfile.UserName == null || Model.userProfile.UserName.Equals(User.GetGitHubLogin()))
```

Comment:
Model.userProfile.UserName won't never be null. right? We should populate default one with username since user is logged in.